brokurly/urls.py

```diff
@@ -1,4 +1,6 @@
 from django.urls   import path, include
 
 urlpatterns = [
+    path('user/', include('user.urls')),
+    path('recipe/', include('recipe.urls')),
```

Comment:
main url에서는 앞 뒤 모두 슬래시를 빼고, app url의 앞에 슬래시를 붙이는 것이 위코드 규칙입니다 👍 
```suggestion
    path('user', include('user.urls')),
    path('recipe', include('recipe.urls')),
```

Reply:
네 수정했습니다~